packages/components/src/__future__/Tag/Tag/_docs/TagMigration.stories.tsx

```diff
@@ -0,0 +1,321 @@
+      <Tag color="red">Tag</Tag>
+      <Tag color="blue">Tag</Tag>
+      <Tag color="yellow">Tag</Tag>
+      <Tag color="orange">Tag</Tag>
+    </div>
+  ),
+}
+
+export const SentimentNone: StoryObj = {
+  render: () => (
+    <Tag
+      color="gray"
+      classNameOverride="bg-white border-default-color border-solid border"
+    >
+      Tag
+    </Tag>
+  ),
+}
+
+export const DismissibleMigration: StoryObj = {
+  render: () => (
+    <RemovableTag
+      removeButtonProps={{
+        ariaLabel: "Remove this tag",
+        onClick: fn(),
+      }}
+    >
+      Tag
+    </RemovableTag>
+  ),
+}
+
+export const AvatarMigration: StoryObj = {
+  render: () => (
+    <div className="flex gap-12">
+      <Tag classNameOverride="ps-4">
+        <span className="flex gap-6 items-center">
+          <Avatar size="small" />
+          Tag
+        </span>
+      </Tag>
+      <Tag classNameOverride="ps-4">
+        <span className="flex gap-6 items-center">
+          <Avatar size="small" fullName="Reed Richards" />
+          Reed Richards
+        </span>
+      </Tag>
+      <Tag classNameOverride="ps-4">
+        <span className="flex gap-6 items-center">
+          <Avatar
+            size="small"
+            fullName="Sue Storm"
+            avatarSrc="https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png"
+          />
+          Susan Storm
+        </span>
+      </Tag>
+    </div>
+  ),
+}
+
+export const AvatarRemovableMigration: StoryObj = {
+  render: () => (
+    <div className="flex gap-12">
+      <RemovableTag
+        removeButtonProps={{
+          ariaLabel: "Remove user from *this context*",
+          onClick: fn(),
+        }}
+        classNameOverride="ps-4"
+      >
+        <span className="flex gap-6 items-center">
+          <Avatar size="small" />
+          Tag
+        </span>
+      </RemovableTag>
+      <RemovableTag
+        removeButtonProps={{
+          ariaLabel: "Remove Reed Richards from *this context*",
+          onClick: fn(),
+        }}
+        classNameOverride="ps-4"
+      >
+        <span className="flex gap-6 items-center">
+          <Avatar size="small" fullName="Reed Richards" />
+          Tag
+        </span>
+      </RemovableTag>
+      <RemovableTag
+        removeButtonProps={{
+          ariaLabel: "Remove Sue Storm from *this context*",
+          onClick: fn(),
+        }}
+        classNameOverride="ps-4"
+      >
+        <span className="flex gap-6 items-center">
+          <Avatar
+            size="small"
+            fullName="Sue Storm"
+            avatarSrc="https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png"
+          />
+          Tag
+        </span>
+      </RemovableTag>
+    </div>
+  ),
+}
+
+export const InlineMigration: StoryObj = {
```

Comment:
I don't think any of these look like optimal implementations? Just one clean example should be enough.

Something like this (assuming gap-12 is the correct size, otherwise whatever it should be according to guidelines).
```tsx
<div className="flex gap-12">
  <Tag>Tag</Tag>
  <Tag>Tag</Tag>
  <Tag>Tag</Tag>
</div>
```

Reply:
When you say optimal implementation, do you mean not in a`ul` with `li`'s? Granted they probably should be since it is a list of tags, or do you mean more use gap as a spacing mechanism?

Reply:
How about something more like:
```
<>
  <ul className="flex m-0 p-0 gap-12 list-none">
    <li>
      <Tag>
        Tag
      </Tag>
    </li>
    <li>
      <Tag>
        Tag
      </Tag>
    </li>
    <li>
      <Tag>
        Tag
      </Tag>
    </li>
  </ul>
</>
```

Reply:
Just the use of `gap-12`. I'd remove the `m-0 p-0` from the snippet since that's related to consumer layout placement.

It's a bit long, so I'd try to reduce the number of lines :) No issues with you using a list though 👍🏻

Reply:
I've updated the examples to just use the gap 12 in the after but had to leave the other tailwind in for the UL. I didn't think we'd need them but looks like our reset doesn't apply to our Stories

Reply:
As much as I like to create semantically correct examples as a sort of guide, I don't think we need to demonstrate it in this case. The focus should be on the `gap` TW class affording the same functionality and we should keep it at that. Especially if we need to remove the focus for other things to be true.

Reply:
I'll strip it back to the basics with Gap and just add a small comment on lists to leave it in our consumers hands